server/asset/polygon/polygon.go

```diff
@@ -0,0 +1,78 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org/license/1.0.0.
+
+package polygon
+
+import (
+	"fmt"
+	"time"
+
+	"decred.org/dcrdex/dex"
+	dexpolygon "decred.org/dcrdex/dex/networks/polygon"
+	"decred.org/dcrdex/server/asset"
+	"decred.org/dcrdex/server/asset/eth"
+)
+
+var registeredTokens = make(map[uint32]*eth.VersionedToken)
+
+func registerToken(assetID uint32, ver uint32) {
+	token, exists := dexpolygon.Tokens[assetID]
+	if !exists {
+		panic(fmt.Sprintf("no token constructor for asset ID %d", assetID))
+	}
+	asset.RegisterToken(assetID, &eth.TokenDriver{
+		DriverBase: eth.DriverBase{
+			Ver: ver,
+			UI:  token.UnitInfo,
+		},
+		Token: token.Token,
+	})
+	registeredTokens[assetID] = &eth.VersionedToken{
+		Token: token,
+		Ver:   ver,
+	}
+}
+
+func init() {
+	asset.Register(BipID, &Driver{eth.Driver{
+		DriverBase: eth.DriverBase{
+			Ver: version,
+			UI:  dexpolygon.UnitInfo,
+		},
+	}})
+
+	registerToken(testTokenID, 0)
+	// registerToken(usdcID, 0)
+
+	if blockPollIntervalStr != "" {
+		blockPollInterval, _ = time.ParseDuration(blockPollIntervalStr)
+		if blockPollInterval < time.Second {
+			panic(fmt.Sprintf("invalid value for blockPollIntervalStr: %q", blockPollIntervalStr))
+		}
+	}
+}
+
+const (
+	BipID              = 966
+	ethContractVersion = 0
+	version            = 0
+)
+
+var (
+	testTokenID, _ = dex.BipSymbolID("dextt.polygon")
```

Comment:
```suggestion
	testTokenID, _ = dex.BipSymbolID("dextt.polygon")
	usdcID, _ = dex.BipSymbolID("usdc.polygon")
```

These suggestions are also in https://github.com/chappjc/dcrdex/commit/69a03e11a270456ef9bc3f26b2ea6a62ca41fa3